src/main/kotlin/biokotlin/cli/Biokotlin.kt

```diff
@@ -0,0 +1,29 @@
+package biokotlin.cli
+
+import biokotlin.util.setupDebugLogging
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+
+/**
+ * This class is the main class for the BioKotlin command line interface.
+ * It is a subclass of CliktCommand and is used to provide users a BioKotlin command line interface.
+ *
+ */
+class Biokotlin : CliktCommand() {
```

Comment:
Should this be `BiokotlinTools` instead? Something similar to the htslib/samtools/bcftools ecosystem.

Reply:
@btmonier Agreed, I changed that